### PR TITLE
Generate RSS feed from markdown news

### PR DESCRIPTION
**Pull request by nethad**
Fixes issue #96 @Nutomic 

An RSS feed for join-lemmy.org news. I took a very pragmatic approach and used JavaScript for it. It parses the markdown news files and generates RSS feed items out of it. During the container build, it generates the feed XML file once and stores it as a static asset to serve as `https://join-lemmy.org/feed.xml`

I hope this is useful and we can get it merged. Let me know if you have any feedback to get this merged.

### CODE REVIEW

File: src/shared/components/main.tsx [hunk 1]
```diff
@@ -490,6 +490,12 @@ export class Main extends Component<Props, State> {
         <InstancePicker reset={this.state.resetInstancePicker} />
         <Helmet title={title}>
           <meta property={"title"} content={title} />
+          <link
+            rel="alternate"
+            type="application/rss+xml"
+            title="RSS Feed for join-lemmy.org"
+            href="/feed.xml"
+          />
```
Comment (nethad) [inline src/shared/components/main.tsx:498]: https://dri.es/rss-auto-discovery

**Review by dessalines — COMMENTED** [review]
Works great, thx! Just two suggestions

File: src/shared/components/news.tsx [hunk 2]
```diff
@@ -45,7 +45,12 @@ function previewMarkdown(markdown: string): string {
 }
 
 const TitleBlock = () => (
-  <div className="pt-16 text-center text-4xl font-bold mb-8">{title}</div>
+  <div className="pt-16 text-center text-4xl font-bold mb-8">
+    {title}
+    <a href="/feed.xml" className="ml-4 inline-block">
+      <img src="/static/assets/images/rss.svg" width={24} />
+    </a>
+  </div>
```
Comment (nethad) [inline src/shared/components/news.tsx:53]: image

**Comment by nethad** [comment]
Thanks for your feedback @dessalines 

Please check out my latest commit and the screenshot attached :+1:

**Review by dessalines — APPROVED** [review]
Perfect thx!

**Pull request merged**